jenkins/common.groovy

```diff
@@ -4,6 +4,18 @@ DRUPAL_MAPPING = [
   'prod': 'vagovprod',
 ]
 
+DRUPAL_ADDRESSES = [
+  'vagovdev'    : 'http://internal-stg-vagovcms-3000-521598752.us-gov-west-1.elb.amazonaws.com',
+  'vagovstaging': 'http://internal-stg-vagovcms-3000-521598752.us-gov-west-1.elb.amazonaws.com',
+  'vagovprod'   : 'http://internal-prod-vagovcms-3000-1370756925.us-gov-west-1.elb.amazonaws.com',
+]
+
+DRUPAL_CREDENTIALS = [
+  'vagovdev'    : 'drupal-dev',
```

Comment:
I guess these are the same and it doesn't matter, but should this say `drupal-staging`?

Reply:
yeah at the moment they are all the same